src/Tribe/Tickets.php

```diff
@@ -513,12 +519,24 @@ public function has_permission( $post, $data, $nonce_action ) {
 				do_action( 'tribe_tickets_ticket_added', $post_id );
 			}
 
-			$return = array( 'html' => $return );
+			parse_str ( $_POST['formdata'], $post_data );
+
+			$ticket = $this->get_ticket( $post_id, $ticket_id );
+			$post_data['ticket_stock'] = $ticket->stock;
+			$post_data['ticket_capacity'] = $ticket->original_stock();
+
+			$return = array(
+				'data' => json_encode( $post_data, JSON_FORCE_OBJECT ),
+			);
+
+			if ( ! empty( $html ) ) {
+				$return['html'] = $html;
+			}
 
 			/**
 			 * Filters the return data for ticket add
 			 *
-			 * @var array Array of data to return to the ajax call
+			 * @param array Array of data to return to the ajax call
```

Comment:
I must have missed this earlier ... but we ought to add a `@param` documenting `$post_id`, too